go/libraries/doltcore/sqle/dsess/autoincrement_tracker.go

```diff
@@ -398,7 +435,20 @@ func (a *AutoIncrementTracker) AcquireTableLock(ctx *sql.Context, tableName stri
 	return a.mm.Lock(tableName), nil
 }
 
-func (a *AutoIncrementTracker) InitWithRoots(ctx context.Context, roots ...doltdb.Rootish) error {
+func (a *AutoIncrementTracker) waitForInit() error {
+	a.wg.Wait()
```

Comment:
Probably want to add a generous timeout here and return a different error if it is surpassed